<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UMessageBoxExClass</class>
 <widget class="QDialog" name="UMessageBoxExClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>147</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UMessageBoxExClass</string>
  </property>
  <widget class="QWidget" name="shadowWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>330</width>
     <height>147</height>
    </rect>
   </property>
   <widget class="QWidget" name="centralWidget" native="true">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>4</y>
      <width>318</width>
      <height>135</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>318</width>
      <height>135</height>
     </size>
    </property>
    <widget class="QLabel" name="messagebox_title">
     <property name="geometry">
      <rect>
       <x>32</x>
       <y>8</y>
       <width>101</width>
       <height>14</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
      </font>
     </property>
     <property name="text">
      <string>有盟装机助手</string>
     </property>
    </widget>
    <widget class="QLabel" name="messagebox_label_1">
     <property name="geometry">
      <rect>
       <x>36</x>
       <y>48</y>
       <width>260</width>
       <height>14</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="messagebox_close">
     <property name="geometry">
      <rect>
       <x>296</x>
       <y>10</y>
       <width>11</width>
       <height>11</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="continue_save">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>104</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>继续保存</string>
     </property>
    </widget>
    <widget class="QPushButton" name="messagebox_cancel">
     <property name="geometry">
      <rect>
       <x>202</x>
       <y>93</y>
       <width>102</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>继续选择软件</string>
     </property>
    </widget>
    <widget class="QLabel" name="messagebox_label_2">
     <property name="geometry">
      <rect>
       <x>36</x>
       <y>68</y>
       <width>260</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../UmInstallerAssistant.qrc"/>
 </resources>
 <connections/>
</ui>
